lib/internal/per_context/domexception.js

```diff
@@ -37,14 +46,39 @@ function ensureInitialized() {
   isInitialized = true;
 }
 
+// This is the same as https://github.com/nodejs/node/blob/12e3c74e2edf78eeae662dce0a9db17a6fe7d730/lib/internal/worker/js_transferable.js#L41-L46.
+// It has been copied here because require() is not available inside this
+// context.
+function makeTransferable(obj) {
+  const inst = ReflectConstruct(JSTransferable, [], obj.constructor);
+  ObjectDefineProperties(inst, ObjectGetOwnPropertyDescriptors(obj));
+  ObjectSetPrototypeOf(inst, ObjectGetPrototypeOf(obj));
+  return inst;
+}
+
 class DOMException extends Error {
   constructor(message = '', name = 'Error') {
     ensureInitialized();
     super();
-    internalsMap.set(this, {
-      message: `${message}`,
-      name: `${name}`
+    const code = nameToCodeMap.get(name) || 0;
+    // We need to define these properties on this throw away object because
+    // makeTransferable() indirectly invokes Error#toString(), which calls the
+    // getters on `this` object but the getters are designed to work only on the
+    // `internals` object.
+    ObjectDefineProperties(this, {
+      message: { configurable: false, enumerable: false,
+                 value: message, writable: false },
+      name: { configurable: false, enumerable: false,
+              value: name, writable: false },
+      code: { configurable: false, enumerable: false,
+              value: code, writable: false }
```

Comment:
```suggestion
      message: {
        configurable: false,
        enumerable: false,
        value: message,
        writable: false
      },
      name: {
        configurable: false,
        enumerable: false,
        value: name,
        writable: false
      },
      code: {
        configurable: false,
        enumerable: false,
        value: code,
        writable: false
      }
```

Reply:
thanks, done in https://github.com/nodejs/node/pull/41148/commits/c5c7858e7b91da39e4defcbc11e9506159e3ae84